src/stories/SegmentedPicker/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { FC, useState } from "react";
+import clsx from "clsx";
+import Text from "../Typography/Text";
+import { SegmentedPickerProps, Segment } from "./type";
+
+export const SegmentedPicker: FC<SegmentedPickerProps<any>> = ({
```

Comment:
please change to T not use any

Reply:
change:
export const SegmentedPicker = <T,>({
  className,
  segments,
  value,
  onSelect,
}: SegmentedPickerProps<T>) => { ...